src/Codata/Musical/Colist/Infinite-merge.agda

```diff
@@ -195,7 +195,7 @@ Any-merge {P = P} xss = mk↔ₛ′ (proj₁ ∘ to xss) from to∘from (proj₂
     ... | inj₂ q | P.refl | q≤p =
       Prod.map there
                (P.cong (there ∘ (Inverse.from (Any-⋎P xs)) ∘ inj₂))
-               (rec (♭ xss , q) (s≤′s q≤p))
+               (rec {- {♭ xss , q} -} (s≤′s q≤p))
```

Comment:
Can remove the comment

Reply:
In which case probably better to leave it in as an implicit so it's type-checked? Otherwise liable to bit-rot!